rootdir/root/init.qcom.rc

```diff
@@ -460,7 +424,6 @@ service qmuxd /system/bin/qmuxd
 
 service netmgrd /system/bin/netmgrd
     class main
-    disabled
```

Comment:
init.class_main.sh do start netmgrd depends on ro.use_data_netmgrd property. We should investigate further why this script isn't run.